src/memray/reporters/flamegraph.py

```diff
@@ -83,22 +211,35 @@ def _from_any_snapshot(
         memory_records: Iterable[MemorySnapshot],
         native_traces: bool,
         temporal: bool,
+        inverted: Optional[bool] = None,
     ) -> "FlameGraphReporter":
-        root: Dict[str, Any] = {
+        # TODO?
+        if temporal and inverted:
+            raise NotImplementedError(
+                "FlamegraphReporter does not support temporal inverted flame graphs."
+            )
+
+        root: FrameNodeDict = {
             "name": "<root>",
-            "location": [html.escape("<tracker>"), "<b>memray</b>", 0],
+            "location": (html.escape("<tracker>"), "<b>memray</b>", 0),
             "value": 0,
             "children": [],
             "n_allocations": 0,
             "thread_id": "0x0",
             "interesting": True,
             "import_system": False,
         }
+        inverted_import_system_root = deepcopy(root) if inverted else None
```

Comment:
I think every variable name in this function with `_import_system_` in the name should be renamed to `_no_import_system_` (or just `_no_imports_`) instead.

`inverted_import_system_root` makes it sound like this is the root of an inverted tree that _does_ contain import system nodes, when actually its the root of an inverted tree that contains only non-import-system nodes.